backend/Contraband/result/views.py

```diff
@@ -248,35 +256,39 @@ def get(self, request, *args, **kwargs):
             # get the data from CSV file
             try:
                 file = read_csv(query.csv_file)
-                try:
-                    if 'user' in request.GET:
-                        user = file[file['fullname'] == request.GET['user']]
-                        if user.empty:
-                            return Response({'message': 'Not Found', 'error': 1}, status=status.HTTP_404_NOT_FOUND)
-                        else:
-                            username, gerrit_username = user.iloc[0, :]['Phabricator'], user.iloc[0, :]['Gerrit']
-                            if user.index[0] != 0:
-                                prev_user = file.iloc[user.index[0]-1, :]['fullname']
-                            else:
-                                prev_user = None
-
-                            if user.index[0] != len(file) - 1:
-                                next_user = file.iloc[user.index[0]+1, :]['fullname']
-                            else:
-                                next_user = None
+                if 'user' in request.GET:
+                    user = file[file['fullname'] == request.GET['user']]
+                    if user.empty:
+                        return Response({'message': 'User not found', 'error': 1}, status=status.HTTP_404_NOT_FOUND)
```

Comment:
This does not show any affect over the app because the frontend of the app is programmed in such a manner so that the it never raises the error. You might have observed the pagination in the app. The error is raised when any developer tries to access the API of the tool and provides with the username that is not in the CSV file.